docs/en/overview/Getting-Started.md

```diff
@@ -23,13 +23,8 @@ The guide will cover the following tasks:
 The optional sections will be labeled with **[Bonus]**.
 
 **Note**  This guide is designed to start an Alluxio system with minimal setup on a single machine.
-
-If you are trying to speedup SQL analytics, you can try the Presto Alluxio Getting Started tutorial:
-
-<p align="center">
-<a href="https://www.alluxio.io/alluxio-presto-sandbox-docker/">
- <img src="https://www.alluxio.io/app/uploads/2019/07/laptop-docker.png" width="250" alt="Laptop with Docker"/></a>
-</p>
+If you are trying to speedup SQL analytics, you can try the 
+[Presto Alluxio Getting Started](https://www.alluxio.io/alluxio-presto-sandbox-docker/) tutorial.
```

Comment:
The picture just says `on laptop with Docker`, which is confusing in this context. We had multiple options here that allowed people to select different scenarios to get started, so the pictures were good to have. Now there is only one option to start with Presto, so the picture is redundant.